src/test/java/org/apache/sysds/test/TestUtils.java

```diff
@@ -1802,6 +1802,16 @@ public static MatrixBlock generateTestMatrixBlock(int rows, int cols, double min
 		return MatrixBlock.randOperations(rows, cols, sparsity, min, max, "Uniform", seed);
 	}
 
+	public static MatrixBlock generateTestMatrixBlockSym(int rows, int cols, double min, double max, double sparsity, long seed){
+		MatrixBlock m = MatrixBlock.randOperations(rows, cols, sparsity, min, max, "Uniform", seed);
+		for(int i = 0; i < rows; i++) {
```

Comment:
consider starting from i = 1

Reply:
If we use i=1 we ignore the first row/col completely.
We changed it and it did not work, maybe we misunderstand